### PR TITLE
change default dnn path on windows

### PR DESCRIPTION
**Pull request by fo40225**
change default dnn path on windows

include path: CUDA_ROOT\include

library path: CUDA_ROOT\lib\x64

#5776

### CODE REVIEW

File: theano/configdefaults.py [hunk 1]
```diff
@@ -390,23 +390,29 @@ def safe_no_dnn_algo_bwd(algo):
...
-    def f(suffix=suffix):
-        if theano.config.cuda.root == '':
-            return ''
-        return os.path.join(theano.config.cuda.root, suffix)
-    return f
+def default_dnn_library_path():
+    if theano.config.cuda.root == '':
+        return ''
+    if sys.platform == 'darwin':
+        return os.path.join(theano.config.cuda.root, 'lib64')
```
Comment (nouiz) [inline theano/configdefaults.py]: The old code was the reverse. on darwin, it is lib and by default, it is lib643

**Comment by nouiz** [comment]
@notoraptor @abergeron what do you think of that?

**Comment by notoraptor** [comment]
It looks good for Windows.

**Comment by abergeron** [comment]
I'm ok with the change, but it will conflict with the PR to remove the old backend.  Can we delay this until then?

File: theano/configdefaults.py (hunk 1 above)
Comment (fo40225) [inline theano/configdefaults.py]: Do you mean "darwin ->lib, linux ->lib64"?

**Comment by nouiz** [comment]
Can you rebase this PR now that the other PR got merged?

thanks

**Comment by nouiz** [comment]
travis have a few failures. Can you fix them? See

https://travis-ci.org/Theano/Theano/jobs/221345146#L873

**Comment by nouiz** [comment]
jenkins test this

**Comment by nouiz** [comment]
thanks

**Pull request merged**